sagemaker_model_governance/model_card_with_model_package.ipynb

```diff
@@ -0,0 +1,426 @@
+   "source": [
+    "model_package_details = ModelPackage.from_model_package_arn(\n",
+    "    model_package_arn=model_pkg[\"ModelPackageArn\"],\n",
+    "    sagemaker_session=sagemaker_session,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "46bc452c-55d8-4395-ad83-7403b2e48d7d",
+   "metadata": {},
+   "source": [
+    "### 2. Initialize a model card\n",
+    "Initialize a model card with the model package details collected in the previous step. When associating model package to a model card, model card will try to auto discover information like training job details and evaluation job details only if there are information like model artifacts and model metrics available in model package. Additionally, it will also try to carry over some additional information like business details to this model card from the previously created the most recent model card that is associated with this particular model package group."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19966fff-86d1-4c06-bc84-05dca057c1ec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_card_name = \"sample-model-card-with-model-package\"\n",
+    "my_card = ModelCard(\n",
+    "    name=model_card_name,\n",
+    "    status=ModelCardStatusEnum.DRAFT,\n",
+    "    model_package_details=model_package_details,\n",
+    "    sagemaker_session=sagemaker_session,\n",
+    ")\n",
+    "my_card.create()\n",
+    "print(f\"Model card {my_card.name} is successfully created with id {my_card.arn}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "fe88e044",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Cleanup (Optional)<a name=\"Cleanup\"></a>\n",
+    "Delete the following resources:\n",
+    "1. The model card\n",
+    "2. The model package\n",
+    "3. The model package group"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d447849",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_card.delete()\n",
+    "\n",
+    "model_pkg = sagemaker_session.sagemaker_client.delete_model_package(\n",
+    "    ModelPackageName=model_pkg[\"ModelPackageArn\"]\n",
+    ")\n",
+    "\n",
+    "sagemaker_session.sagemaker_client.delete_model_package_group(\n",
+    "    ModelPackageGroupName=model_package_group_name\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ba5c2fdd",
+   "metadata": {},
+   "source": [
+    "## Notebook CI Test Results\n",
+    "\n",
+    "This notebook was tested in multiple regions. The test results are as follows, except for us-west-2 which is shown at the top of the notebook.\n",
+    "\n",
+    "\n",
+    "![This us-east-1 badge failed to load. Check your device's internet connectivity, otherwise the service is currently unavailable](https://h75twx4l60.execute-api.us-west-2.amazonaws.com/sagemaker-nb/us-east-1/NOTEBOOK_PATH)\n",
```

Comment:
update `NOTEBOOK_PATH` to `sagemaker_model_governance|model_card_with_model_package.ipynb`